shopelectro/tests/tests_selenium.py

```diff
@@ -283,19 +295,15 @@ def test_add_to_cart(self):
         """We can add item to cart from it's category page."""
         self.browser.get(self.children_category)
         self.wait_page_loading()
-        self.browser.find_elements_by_class_name(
-            'js-product-to-cart')[0].click()
+        self.add_to_cart(0)
```

Comment:
might be more clear, but decide

```suggestion
        self.add_to_cart(index=0)
```